secure/key-manager/src/lib.rs

```diff
@@ -288,6 +293,19 @@ where
             }
         }
     }
+
+    fn get_operator_account(&self) -> Result<AccountAddress, Error> {
+        match self.storage.get(OPERATOR_ACCOUNT) {
+            Ok(response) => match response.value.string() {
+                Ok(account_address) => match AccountAddress::from_str(&account_address) {
+                    Ok(account_address) => Ok(account_address),
+                    Err(e) => Err(Error::UnknownError(e.to_string())),
+                },
+                Err(e) => Err(Error::MissingAccountAddress(e)),
+            },
+            Err(e) => Err(Error::MissingAccountAddress(e)),
+        }
```

Comment:
Have you played aorund with map or and_then?

Reply:
Thanks, David! I managed to simplify it a little (removed one match statement), but can't remove the inner match statement because of the different error type.. it's also unnecessarily ugly because we can't convert directly from anyhow.. but such is life!